src/utils/matching/mergeMasks.ts

```diff
@@ -0,0 +1,16 @@
+import { pathToRegExp } from 'node-match-path'
+import { Mask } from '../../setupWorker/glossary'
+
+export function mergeMasks(base: string, mask: Mask): Mask {
+  const normalizedBase = base.replace(/\/$/, '')
+
+  if (typeof mask === 'string') {
+    try {
+      return new URL(mask, base).href
```

Comment:
Convert to `URL` and return its `href` to normalize two strings (double slashes, hashes, etc.).